Jenkinsfile

```diff
@@ -16,20 +16,23 @@ def baseConfig = {
 
 def defaultParams = [
     booleanParam(name: 'RELEASE_BUILD',
-        description: 'Is this a release build.',
+        description: 'Is this a release build. If so, GIT_REVISION must be specified, and only on-prem images will be built.',
```

Comment:
when you say only on-prem images will be built you mean it won't trigger the cloud build, correct? because this build only builds on-prem images anyways, right. maybe we can change this to just say won't trigger cloud build because it makes it seem like this job could actually do that. i am thinking this wording is probably just left over from the original draft where this was doing both builds.

Reply:
Yeah, leftover from an earlier version as you said. I've updated this.